docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,73 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_Feynman Zhou, CNCF Ambassador, Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the CLI tool on Aug 22. ORAS v0.14 provides four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provide new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS v0.14 to copy an image from a validated artifact registry to personal conintainer registry, then attach a SBOM to it and discover the graph.
+
+
+## Install ORAS 0.14
+
+Install the latest release of ORAS on a Linux machine:
```

Comment:
This solves linux, should we point to a common install oras page in oras.land?
And, when, oh when can we apt-get/win-get install :)